sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/GotoStatement.vb

```diff
@@ -0,0 +1,28 @@
+﻿Imports System
+
+Namespace Tests.Diagnostics
+
+    Public Class GotoStatement
+
+        Sub GotoStatement(condition As Boolean)
+
+            If stubid Then
+                GoTo Label 'Noncompliant
```

Comment:
Please add one precise location verification as well. I forgot this one before. Since it's at the beginning of the line, you cannot use `// ^^^^^` here. Use this instead `' Noncompliant ^17#4` to mark start and length (I'm not sure if I counted the begin index correctly).